packages/ipfs-http-client/package.json

```diff
@@ -18,7 +18,9 @@
   "exports": {
     ".": {
       "import": "./src/index.js"
-    }
+    },
+    "./index.js": "./index.js",
+    "./index.min.js": "./index.min.js"
```

Comment:
These files [don't exist in the published package](https://unpkg.com/browse/ipfs-http-client@57.0.3/) - are you sure this patch solves the issue?

Reply:
The build output seems to have changed since 56.0.3 so I will re-investigate and see if I can fix the issue without the need for the package.json patch.